recipes/acado/all/cmake/qpoases.cmake

```diff
@@ -0,0 +1,17 @@
+SET( ACADO_QPOASES_EMBEDDED_SOURCES 
```

Comment:
Can you please explain the purpose for these files to be linked? Are you trying to override/correct the list in the upstream sources?

Reply:
Yes if course. Those files are required for the generated code to work. They are a customized version of qpOASES, targeted for higher performance for embedded usage (hence the name). I do not know of any other way to provide those files, to the consumer similar to how the original package does it. See also: https://cpplang.slack.com/archives/C41CWV9HA/p1608543272079800